scripts/rollup/bundles.js

```diff
@@ -125,6 +125,34 @@ const bundles = [
       'src/shared/**/*.js',
     ],
   },
+  {
+    babelOpts: babelOptsReact,
+    bundleTypes: [NODE_DEV],
+    config: {
+      destDir: 'build/',
+      globals: {
+        react: 'React',
+      },
+      moduleName: 'ReactTestUtils',
+      sourceMap: false,
+    },
+    entry: 'src/test/ReactTestUtils.js',
+    externals: ['prop-types', 'prop-types/checkPropTypes', 'react'],
+    hasteName: 'ReactTestUtils',
+    isRenderer: false,
+    label: 'react-test-utils',
+    manglePropertiesOnProd: false,
+    name: 'react-test-utils',
+    paths: [
+      'src/renderers/dom/**/*.js',
+      'src/renderers/shared/**/*.js',
+      'src/test/**/*.js', // ReactTestUtils is currently very coupled to DOM.
+
+      'src/isomorphic/classic/types/checkPropTypes.js',
```

Comment:
That is a bit silly, now that you point it out. It was unintentional. `src/isomorphic/classic/types/checkPropTypes.js` just forwards to `prop-types/checkPropTypes`. But I can remove the unnecessary indirection. 😄